### PR TITLE
Implement test utility for testing imported events

### PR DESCRIPTION
**Pull request by vlad-lubenskyi**
In this PR we:
 - added `importEvent()` to `AggregateTestSupport` (and thus to `AggregateMessageDispatcher`);
 - implemented `AggregateEventImportTest` and `EventApplierExpected` for testing imported events;
 - covered `Aggregate...Test` utilities working with aggregate parts.

### CODE REVIEW

File: testutil-server/src/main/java/io/spine/testing/server/aggregate/AggregatePartEventReactionTest.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    protected abstract R newRoot(I id);
+
+    /**
+     * Instantiates a new aggregate part with the given root.
+     *
+     * <p>A typical implementation:
+     * <pre>
+     *     {@code
+     *     \@Override
+     *     protected MyAggregatePart newRoot(MyAggregateRoot root) {
```
Comment (armiol) [inline testutil-server/src/main/java/io/spine/testing/server/aggregate/AggregatePartEventReactionTest.java]: Copy-paste detected.

File: testutil-server/src/test/java/io/spine/testing/server/aggregate/given/SamplePartEventImportTest.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    }
+
+    @Override
+    protected Repository<TuTaskId, TuAggregatePart> createEntityRepository() {
+        return new TuAggregatePartRepository();
+    }
+
+    @Override
+    public EventReactorExpected<TuComments>
+    expectThat(TuAggregatePart entity) {
```
Comment (armiol) [inline testutil-server/src/test/java/io/spine/testing/server/aggregate/given/SamplePartEventImportTest.java]: I think it fits the previous line.

File: testutil-server/src/test/java/io/spine/testing/server/aggregate/given/SampleEventImportTest.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+ */
+public class SampleEventImportTest
+        extends AggregateEventImportTest<TuProjectId,
+                                         TuTrelloProjectCreated,
+                                         TuProject,
+                                         TuAggregate> {
+
+    public static final TuTrelloProjectCreated TEST_EVENT =
+            TuTrelloProjectCreated.newBuilder()
+                            .setId(TuAggregate.ID)
```
Comment (armiol) [inline testutil-server/src/test/java/io/spine/testing/server/aggregate/given/SampleEventImportTest.java]: Looks misaligned.

File: testutil-server/src/test/java/io/spine/testing/server/aggregate/given/SampleEventImportTest.java [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+import io.spine.testing.server.aggregate.given.agg.TuAggregateRepository;
+import io.spine.testing.server.expected.EventReactorExpected;
+import io.spine.testing.server.given.entity.TuProject;
+import io.spine.testing.server.given.entity.TuProjectId;
+import io.spine.testing.server.given.entity.event.TuTrelloProjectCreated;
+
+/**
+ * The test class for checking an import of an event into an aggregate.
+ *
+ * @author Vladyslav Lubenskyi
```
Comment (armiol) [inline testutil-server/src/test/java/io/spine/testing/server/aggregate/given/SampleEventImportTest.java]: We don't use `@author` tags anymore to make some emphasis on a collective code ownership. 

Spine will get rid of `@author`s at all soon. So please remove from the files modified in this changeset.

File: testutil-server/src/test/java/io/spine/testing/server/aggregate/given/SampleEventImportTest.java [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+        super(TuAggregate.ID, TEST_EVENT);
+    }
+
+    @Override
+    protected Repository<TuProjectId, TuAggregate> createEntityRepository() {
+        return new TuAggregateRepository();
+    }
+
+    @Override
+    public EventReactorExpected<TuProject> expectThat(TuAggregate entity) {
```
Comment (armiol) [inline testutil-server/src/test/java/io/spine/testing/server/aggregate/given/SampleEventImportTest.java]: Looking at this sample usage of the test helper class I understand nothing. 

As far as I know, an event import should cause the change in the aggregate state. However, I don't understand how this is tested, as you just call `super`.

File: testutil-server/src/test/java/io/spine/testing/server/aggregate/given/SamplePartEventReactionTest.java [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+    @Override
+    protected TuAggregateRoot newRoot(TuTaskId id) {
+        return TuAggregateRoot.newInstance(id);
+    }
+
+    public TuReactingAggregatePart createPart(TuTaskId id) {
+        return newPart(id);
+    }
+
```
Comment (armiol) [inline testutil-server/src/test/java/io/spine/testing/server/aggregate/given/SamplePartEventReactionTest.java]: Please kill this empty line.

File: testutil-server/src/test/java/io/spine/testing/server/aggregate/given/SamplePartEventReactionTest.java [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+        return new TuReactingAggregatePartRepository();
+    }
+
+    @Override
+    public EventReactorExpected<TuComments>
+    expectThat(TuReactingAggregatePart entity) {
+        return super.expectThat(entity);
+    }
+
+    public Message storedMessage() {
```
Comment (armiol) [inline testutil-server/src/test/java/io/spine/testing/server/aggregate/given/SamplePartEventReactionTest.java]: If it isn't an override, then I don't understand why this method exists.

File: testutil-server/src/test/java/io/spine/testing/server/aggregate/given/agg/TuReactingAggregatePartRepository.java [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+import io.spine.server.aggregate.AggregatePartRepository;
+import io.spine.testing.server.given.entity.TuTaskId;
+
+/**
+ * @author Vladyslav Lubenskyi
+ */
+public final class TuReactingAggregatePartRepository
+        extends AggregatePartRepository<TuTaskId, TuReactingAggregatePart, TuAggregateRoot> {
+
```
Comment (armiol) [inline testutil-server/src/test/java/io/spine/testing/server/aggregate/given/agg/TuReactingAggregatePartRepository.java]: Please kill this line.

File: testutil-server/src/test/java/io/spine/testing/server/aggregate/given/agg/TuReactingAggregatePart.java [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+    @React
+    TuCommentLimitReached on(TuCommentAdded event) {
+        return TuCommentLimitReached.newBuilder()
+                                    .setId(event.getId())
+                                    .build();
+    }
+
+    @Apply
+    void on(TuCommentLimitReached event) {
+        getBuilder().setTimestamp(fromMillis(123));
```
Comment (armiol) [inline testutil-server/src/test/java/io/spine/testing/server/aggregate/given/agg/TuReactingAggregatePart.java]: Please make the implementation looking more real to what's supposed to happen upon `TuCommentLimitReached`.

File: version.gradle [hunk 10]
```diff
@@ -25,7 +25,7 @@
  *  as we want to manage the versions in a single source.
  */
 
-def final SPINE_VERSION = '0.11.00-SNAPSHOT'
+def final SPINE_VERSION = '0.11.01-SNAPSHOT'
```
Comment (armiol) [inline version.gradle]: Please make it with no zero.  The next version would be `0.11.1-SNAPSHOT`. And if Dima already merged to `master`, this changeset should be an increment (e.g. `0.11.2-SNAPSHOT — please recheck).

**Comment by vlad-lubenskyi** [comment]
@armiol PTAL

File: testutil-server/src/test/java/io/spine/testing/server/aggregate/given/SampleEventImportTest.java [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+    }
+
+    @Override
+    protected Repository<TuProjectId, TuAggregate> createEntityRepository() {
+        return new TuAggregateRepository();
+    }
+
+    @Override
+    @VisibleForTesting
+    public EventReactorExpected<TuProject> expectThat(TuAggregate entity) {
```
Comment (armiol) [inline testutil-server/src/test/java/io/spine/testing/server/aggregate/given/SampleEventImportTest.java]: As discussed, please rethink the `EventReactorExpected` usage in a sample of event _import_ test.

**Comment by vlad-lubenskyi** [comment]
@armiol PTAL

**Pull request merged**